frontend/public/components/pod-exec.jsx

```diff
@@ -36,6 +37,7 @@ export class PodExec extends React.PureComponent {
   connect_ () {
     const { metadata } = this.props.obj;
     const { activeContainer } = this.state;
+    const usedClient = connectToFlags(FLAGS.OPENSHIFT) ? 'oc' : 'kubectl';
```

Comment:
You need to wrap a React component with `connectToFlags()`, in this case:
```javascript
export const PodExec = connecToFlags(FLAGS.OPENSHIFT)(class PodExec extends React.PureComponent {
  ...
  const usedClient = this.props.flags[FLAGS.OPENSHIFT] ? 'oc' : 'kubectl';
  ...
});
```
See https://github.com/openshift/console/blob/master/frontend/public/components/cluster-overview.jsx#L261